src/main/kotlin/de/bund/digitalservice/useid/identification/IdentificationSessionsController.kt

```diff
@@ -93,21 +96,40 @@ class IdentificationSessionsController(
             }
             .defaultIfEmpty(ResponseEntity.status(HttpStatus.NOT_FOUND).body(null))
             .doOnError { exception ->
-                log.error { "error occurred while getting the tc token for session with id $useIDSessionId; ${exception.message}" }
+                log.error { "error occurred while getting tc token for useIDSessionId $useIDSessionId;\n ${exception.message}" }
             }
             .onErrorReturn(
                 ResponseEntity.internalServerError().body(null)
             )
     }
 
     @GetMapping("/{eIDSessionId}")
```

Comment:
maybe we should extend the path?
`GET /api/v1/identification/sessions/{eIDSessionId}/identities`
`GET /api/v1/identification/sessions/{useIDSessionid}/identities/{eIDSessionId}`

Github has amazing public endpoints, maybe we could have it as a reference: https://docs.github.com/en/rest/overview/endpoints-available-for-github-apps

Reply:
Extracted this to new ticket: 
https://digitalservicebund.atlassian.net/browse/USEID-406